.github/workflows/test.yml

```diff
@@ -182,6 +182,26 @@ jobs:
             tox_env: "doctesting"
             use_coverage: true
 
+    continue-on-error: >-
+      ${{
+        contains(
+          fromJSON(
+            '[
+              "windows-py38-pluggy",
+              "windows-py313",
+              "ubuntu-py38-pluggy",
+              "ubuntu-py38-freeze",
+              "ubuntu-py313",
+              "macos-py38",
+              "macos-py313"
```

Comment:
This means that this needs to be manually updated, when interpreters and Osses are changing. I guess we do not have some single source of truth for that and if we had this file would need to be generated, correct?

Reply:
This place *is* the source of truth.